spec/memo_wise_spec.rb

```diff
@@ -351,11 +351,32 @@ def module2_method
           end
         end
 
+        let(:klass_with_initializer) do
+          Class.new do
+            include Module1
+            def initialize(*); end
```

Comment:
Curious why `*` instead of `...` -- apologies I really do not remember the difference, just curious

Reply:
I couldn't find a reason to use arguments at all (and confirmed the test passes without them but fails without our code change) so I removed them.